src/pks/flow/constitutive_relations/sources/transpiration_distribution_evaluator.cc

```diff
@@ -235,11 +235,11 @@ TranspirationDistributionEvaluator::EnsureCompatibility(const Teuchos::Ptr<State
 bool
 TranspirationDistributionEvaluator::TranspirationPeriod_(double time) {
   double time_of_year = fmod(time, year_duration_);
-  if (leaf_on_time_ <= time < leaf_off_time_) {
+  if (leaf_on_time_ <= time_of_year && time_of_year < leaf_off_time_) {
     return true;
   } else if (leaf_on_time_ < leaf_off_time_) {
     return false;
-  } else if (leaf_off_time_ <= time < leaf_on_time_) {
+  } else if (leaf_off_time_ <= time_of_year || time_of_year < leaf_on_time_) {
```

Comment:
Shouldn't this be:

    } else if (leaf_off_time_ <= time_of_year && time_of_year < leaf_on_time_) {
      return false;  // winter in the southern hemisphere
    } else {
      return true;
    }

This is southern hemisphere (where leaf off is in e.g. March and leaf on is in e.g. September).